src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/crafting/AnvilListener.java

```diff
@@ -34,7 +34,7 @@ public void onAnvil(InventoryClickEvent e) {
 
             if (hasUnallowedItems(item1, item2)) {
                 e.setResult(Result.DENY);
-                Slimefun.getLocalization().sendMessage((Player) e.getWhoClicked(), "anvil.not-working", true);
+                Slimefun.getLocalization().sendMessage(e.getWhoClicked(), "anvil.not-working", true);
```

Comment:
This cast cannot be removed due to backwards compatibility.

Reply:
Good to know, I did wonder why this was as it is - thanks